playbooks/ansible-security-trendmicro-deepsec-appliance/roles/tm-deepsec-aws-setup/tasks/main.yml

```diff
@@ -0,0 +1,204 @@
+---
+# tasks file for tm_silent_setup
+
+- name: Ensure bash, OpenSSl, and libssl are the latest versions
+  become: true
+  yum:
```

Comment:
lets use package

Reply:
these package tasks, could be moved into single instances with packages as a list